dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/dispatcher/internal/channel/HttpDispatcherLink.java

```diff
@@ -354,6 +360,15 @@ public void ready(VirtualConnection inVC) {
         this.isc = (HttpInboundServiceContextImpl) getDeviceLink().getChannelAccessor();
         this.remoteAddress = isc.getRemoteAddr();
 
+        //Add for Servlet 6.0
+        //HttpDispatcherLink can be reused but ready(VirtualConnection) is always called to get a current VirtualConnection.
+        //If thats true, don't need to clean up this connectionID
```

Comment:
Is it true or do we need some cleanup?

Reply:
I will need @mrsaldana assessment here.  From what I see, the HttpDispatcherLink.ready(VirtualConnection) is always called for each request.  If that is true, there is no cleanup needed all variables inside this ready() will be reinitialized.

Reply:
May need to test with the keepconnectionalive header to see if the connectionID get changed.